std_msgs/String[] objects
